Refresh AITK Model Catalog Page (#8462)
Microsoft/vscode-docs: docs/intelligentapps/models.md

1. The flow for adding models will be slightly different based on the providers:

    - **GitHub**: AI Toolkit will ask for your GitHub credentials to access the model repository. Once authenticated, the model will be added directly into AI Toolkit.
    - **Azure AI Foundry**: AI Toolkit will prompt you to sign in with your Azure account and set a default project in Azure AI Foundry. Once authenticated, you can deploy a model to your Azure AI Foundry project and use it in AI Toolkit.
    - **ONNX**: The model is downloaded from ONNX and added to AI Toolkit.
    - **Ollama**: The model is downloaded from Ollama and added to AI Toolkit.

    > [!TIP]
    > You can edit the API key later by right clicking the model and selecting **Edit** and view the excrypted value in `${HOME}/.aikt/models/my-models/yml` file.

    ![AI Toolkit interface showing a model card with options Try in Playground, Download, and Load in Playground.](./images/models/model_operation.png)

    - **OpenAI**, **Anthropic**, and **Google**: AI Toolkit will prompt you to enter the API Key.
    - **Custom models**: Refer to the [Add a custom model](#add-a-custom-model) section for detailed instructions.

> Once added, the model will appear under **MY MODELS** in the tree view, and you can use it in the [**Playground**](/docs/intelligentapps/playground.md) or [**Agent Builder**](/docs/intelligentapps/agentbuilder.md).

Reviewer: ```suggestion
Once added, the model appears under **MY MODELS** in the tree view, and you can use it in the [**Playground**](/docs/intelligentapps/playground.md) or [**Agent Builder**](/docs/intelligentapps/agentbuilder.md).
```

## Deploy a model to Azure AI Foundry
If you want to deploy a model to Azure AI Foundry, you can do so directly from the model catalog. This allows you to run the model in your Azure AI Foundry project and use it in AI Toolkit.
1. Locate the model you want to deploy in the model catalog

> 1. Select the **Deploy to Azure** button on the model card

Reviewer: ```suggestion
1. Select **Deploy to Azure** on the model card
```

1. AI Toolkit will prompt you to sign in with your Azure account and set a default project in Azure AI Foundry
1. Once authenticated, you can deploy the model to your Azure AI Foundry project
1. After deployment, the model will be available in your Azure AI Foundry project and can be used in AI Toolkit

![AI Toolkit interface showing a model card with options Deploy to Azure, Try in Playground, and Load in Playground.](./images/models/foundry_models.png)

## Add a custom model
You can also add your own models that are hosted externally or run locally. There are several options available:
- Add Ollama models from the Ollama library or custom Ollama endpoints.

> - Add custom models that has an OpenAI compatible endpoint, such as a self-hosted model or a model running on a cloud service.

Reviewer: ```suggestion
- Add custom models that have an OpenAI compatible endpoint, such as a self-hosted model or a model running on a cloud service.
```

- Add custom ONNX models, such as those from Hugging Face, using AI Toolkit's [model conversion tool](/docs/intelligentapps/modelconversion.md).

There are several entrypoints to add models to AI Toolkit:
- From **MY MODELS** in the tree view, hover over it and select the `+` icon.

> ![AI Toolkit interface showing the Model Catalog toolbar with the + Add model button highlighted, indicating where users can click to add a new custom model.](./images/models/custom_1.png)

Author: Updated

- From the **Model Catalog**, select the **+ Add model** button from the tool bar.

> ![AI Toolkit interface showing the Model Catalog toolbar with the + Add model button highlighted. The toolbar is located at the top of the catalog view, and the + Add model button is emphasized to indicate where users can click to add a new custom model.](./images/models/custom_2.png)

Reviewer: Fix indentation

- From the **Add Custom Models** section in the model catalog, select **+ Add Your Own Model**.

> ![AI Toolkit interface showing the Custom Models section in the model catalog. The + Add model button is highlighted, indicating where users can click to add a new custom model.](./images/models/custom_3.png)

Author: Updated
